package.json

```diff
@@ -146,7 +146,10 @@
   ],
   "codegenConfig": {
     "name": "rngesturehandler_codegen",
-    "type": "components",
-    "jsSrcsDir": "./src/specs"
+    "type": "all",
+    "jsSrcsDir": "./src/specs",
```

Comment:
Sometimes we use `fabric` and sometimes we use `specs` 😅  But `specs` sound better since turbomodules are not fabric.

Reply:
What do you mean? I think the only other place where `fabric` appears is inside `android` directory and contains code specific to the new-arch. I guess we could rename that to `newarch` and `paper` to `oldarch`. LMK what you think.

Reply:
I meant `./src/specs` where in e.g. `RNScreens` we have https://github.com/software-mansion/react-native-screens/tree/336efc0c715c1312ec4f9d8bd5709cc1d884709b/src/fabric

Reply:
Ah, it was `fabric` here as well but I changed it in https://github.com/software-mansion/react-native-gesture-handler/pull/2461.

`specs` makes way more sense I think - as you noted, turbomodules are not fabric.